src/pages/Home/Home.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import getToken from '../../lib/tokens';
+
+const Home = () => {
+  const saveToken = () => {
+    const token = getToken();
+    localStorage.setItem('token', token);
+    window.location.reload(false);
```

Comment:
From what I could find out passing in the flag of 'false' is the aspect that is deprecated. As it turns out, the default value for `reload()` is false so there is no need to pass that in. 

With that said, I generally don't use `window.location` methods unless I need to send the user to an external URL.

In this instance, rather than reload the page, thus letting the check for the token cause the redirect to the `/list` route, I would recommend using the options afforded to you by `react-router`. 

In particular you might want to look at the `useHistory` hook from react-router. You can learn more about that hook [HERE](https://reactrouter.com/web/api/Hooks/usehistory).